Application/EdFi.Ods.AdminApi/E2E Tests/gh-action-setup/.automation.env

```diff
@@ -14,8 +15,7 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 PGBOUNCER_LISTEN_PORT=6432
 
-# The following needs to be set to specify the ODS API endpoint for Admin API to internally connect.
-# If user chooses direct connection between ODS API and Admin API within docker network, then set the api internal url as follows
-API_INTERNAL_URL=http://${ODS_VIRTUAL_NAME}
+# For Supported ODS API Version
+ODS_API_VERSION=${ODS_API_VERSION}
```

Comment:
@simpat-adam 
Defining "ODS_API_VERSION" once should be enough. Line 19 can be removed. Please consider changing all the .env files.